geode-core/src/main/java/org/apache/geode/internal/cache/event/DistributedEventTracker.java

```diff
@@ -265,7 +272,13 @@ public void recordEvent(InternalCacheEvent event) {
       canonicalizeIDs(tag, v);
     }
 
-    EventSequenceNumberHolder newEvh = new EventSequenceNumberHolder(eventID.getSequenceID(), tag);
+    Object key = null;
+    if (event instanceof EntryEventImpl) {
+      key = ((EntryEventImpl) event).getKey();
+    }
+
+    EventSequenceNumberHolder newEvh =
```

Comment:
Please avoid abbreviations and spell out a full variable name that means something like `eventSequence` or `eventSequenceId`.